run_dir/design/base.html

```diff
@@ -51,30 +51,34 @@
-                    <ul class="dropdown-menu" role="menu">
-                        <li><a href="/applications">Application Counts</a></li>
-                        <li><a href="/production/cronjobs">Cronjobs List</a></li>
-                        <li><a href="/nas_quotas">NAS Quotas</a></li>
-                        <li><a href="/flowcells_plot">Flowcells Trends</a></li>
-                        <li><a href="/flowcell_count_plot">Flowcells Counts</a></li>
-                        <li><a href="/reads_total/">Read Count Totals</a></li>
-                        <li><a href="/proj_meta">Compare Project Meta</a></li>
-                        <li><a href="/instrument_logs">View Instrument Logs</a></li>
-                        <li><a href="/workset_placement">Workset Setup</a></li>
-                        <li><a href="http://lims-dashboard.scilifelab.se">Script Dashboard</a></li>
+                    <ul class="dropdown-menu" role="menu"><li class="dropdown-header">General stats</li>
+                    <li><a href="/applications">Application Counts</a></li>
+                    <li><a href="/production/cronjobs">Cronjobs List</a></li><li role="separator" class="divider"></li>
+                    <li class="dropdown-header">Flowcells</li>
+                    <li><a href="/flowcells_plot">Flowcells Trends</a></li>
+                    <li><a href="/flowcell_count_plot">Flowcells Numbers</a></li><li role="separator" class="divider"></li>
+                    <li class="dropdown-header">Tools</li>
+                    <li><a href="/nas_quotas">NAS Quotas</a></li>
+                    <li><a href="/reads_total/">Read Count Totals</a></li>
+                    <li><a href="/proj_meta">Compare Project Meta</a></li>
+                    <li><a href="/instrument_logs">View Instrument Logs</a></li>
+                    </li><li role="separator" class="divider"></li>
+                    <li><a href="http://lims-dashboard.scilifelab.se">Script Dashboard</a></li>
                     </ul>
-                </li>
-                <li class="dropdown" id="projects">
+                    </li>
+                    <li class="dropdown" id="projects">
                     <a class="dropdown-toggle" data-toggle="dropdown" href="#projects">Projects <span class="caret"></span></a>
                     <ul class="dropdown-menu" role="menu">
-                        <li><a href="/projects/all">All</a></li>
-                        <li><a href="/projects/pending">Pending</a></li>
-                        <li><a href="/projects/reception_control">Reception Control</a></li>
-                        <li><a href="/projects/ongoing">Ongoing</a></li>
-                        <li><a href="/projects/pending_review">Needs Review</a></li>
-                        <li><a href="/projects/closed">Closed</a></li>
-                        <li><a href="/projects/aborted">Aborted</a></li>
-                    </ul>
+                  <li><a href="/projects/all">All</a>
```

Comment:
Indentation again 😄 Sorry, I keep thinking that the HTML is messed up then realise that it's fine, which is why I've mentioned it each time.

Reply:
sorry, once again I'm just being sloppy!